task/servicetest/servicetest.go

```diff
@@ -110,6 +110,13 @@ type System struct {
 func testTaskCRUD(t *testing.T, sys *System) {
 	orgID := idGen.ID()
 	userID := idGen.ID()
+	if sys.CredsFunc != nil {
+		var err error
+		orgID, userID, _, err = sys.CredsFunc()
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
 
 	// Create a task.
 	task := &platform.Task{OrganizationID: orgID, Owner: platform.User{ID: userID}, Flux: fmt.Sprintf(scriptFmt, 0)}
```

Comment:
I don't necessarily see the path forward here, with omitting the owner from the response but still accepting it in the create arguments.

I understand that https://github.com/influxdata/influxdb/issues/11427 is to track the big picture, and if work needs to be pushed there, that's fine.

Should this test be updated to check the owner/member API to ensure that the owner matches what was provided? Or would that assertion be part of https://github.com/influxdata/influxdb/issues/11427?

Reply:
You're right. I think that all TaskCreates should only be able to accept the owner through the context and not in the task struct. It is strange to me that you can set the owner of a task as someone other than yourself.

Reply:
After our conversation I thing this is going to be removed in a later pr and is left here until we can make other changes in the backend.